### PR TITLE
Download page updates for NetBeans 22 release

### PR DESCRIPTION
**Pull request by mbien**
download page updates for NB 22

### CODE REVIEW

File: modules/ROOT/pages/download/nb22/index.adoc [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+. Follow the build instructions in the README
+
+== Community Approval
+
+As in any other Apache Project, the Apache NetBeans Community approved this release
+through the following voting processes in our xref:community/mailing-lists.adoc[mailing lists]:
+
+- link:https://lists.apache.org/thread/rvw8f43lgbpjmpqmdwq1cbkcnpf3ryyv[PMC vote]
+// TODO update link once available
+- link:https://lists.apache.org/thread/jzhcxv8jczl83g01vlzlxfpgjb5fk8yl[PMC vote result]
```
Comment (mbien) [inline modules/ROOT/pages/download/nb22/index.adoc]: second link needs update once the thread is available - i can force push

**Comment by ebarboni** [comment]
I think we can merge, the banner is on antora ui so we can test the page live without too many advertisement yet

**Comment by mbien** [comment]
https://netbeans.apache.org/download/nb22/ doesn't seem to work yet

**Comment by ebarboni** [comment]
well jenkins seems not in good shape. main build and antora build are wrong with git.

**Pull request merged**